teamcity: failed tests on master: testrace/TestBackupRestoreSystemJobsProgress

The following tests appear to have failed:

[#300397](https://teamcity.cockroachdb.com/viewLog.html?buildId=300397):

```
--- FAIL: testrace/TestBackupRestoreSystemJobsProgress (0.000s)
Race detected!

------- Stdout: -------
W170719 18:40:40.838101 50313 server/server.go:299  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170719 18:40:40.839384 50313 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170719 18:40:40.868472 50313 server/config.go:534  [n?] 1 storage engine initialized
I170719 18:40:40.868705 50313 server/config.go:536  [n?] RocksDB cache size: 512 MiB
I170719 18:40:40.868755 50313 server/config.go:536  [n?] store 0: in-memory, size 100 MiB
I170719 18:40:40.870446 50313 server/node.go:434  [n?] store [n0,s0] not bootstrapped
I170719 18:40:40.897017 50313 server/node.go:369  [n?] **** cluster a185a4c7-e0ed-40fa-a7cd-3d3d21284208 has been created
I170719 18:40:40.897146 50313 server/node.go:370  [n?] **** add additional nodes by specifying --join=127.0.0.1:60206
I170719 18:40:40.917239 50313 storage/store.go:1260  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170719 18:40:40.940628 50313 server/node.go:447  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:1 WritesPerSecond:47.95243003854344}
I170719 18:40:40.940942 50313 server/node.go:331  [n1] node ID 1 initialized
I170719 18:40:40.941190 50313 gossip/gossip.go:297  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:60206" > attrs:<> locality:<> 
I170719 18:40:40.942073 50313 storage/stores.go:295  [n1] read 0 node addresses from persistent storage
I170719 18:40:40.942377 50313 server/node.go:588  [n1] connecting to gossip network to verify cluster ID...
I170719 18:40:40.942524 50313 server/node.go:613  [n1] node connected via gossip and verified as part of cluster "a185a4c7-e0ed-40fa-a7cd-3d3d21284208"
I170719 18:40:40.947400 50313 server/node.go:385  [n1] node=1: started with [=] engine(s) and attributes []
I170719 18:40:40.980329 50411 storage/replica_command.go:2673  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
E170719 18:40:41.016572 50412 storage/queue.go:658  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170719 18:40:41.020409 50411 storage/replica_command.go:2673  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/tsd [r3]
I170719 18:40:41.022860 50313 sql/executor.go:364  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:60206}
I170719 18:40:41.070753 50313 server/server.go:815  [n1] starting https server at 127.0.0.1:34285
I170719 18:40:41.070878 50313 server/server.go:816  [n1] starting grpc/postgres server at 127.0.0.1:60206
I170719 18:40:41.070923 50313 server/server.go:817  [n1] advertising CockroachDB node at 127.0.0.1:60206
E170719 18:40:41.102462 50412 storage/queue.go:658  [replicate,n1,s1,r2/1:/System/{-tsd}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170719 18:40:41.121193 50411 storage/replica_command.go:2673  [split,n1,s1,r3/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r4]
E170719 18:40:41.219859 50543 storage/replica_proposal.go:522  [n1,s1,r3/1:/{System/tsd-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170719 18:40:41.229615 50411 storage/replica_command.go:2673  [split,n1,s1,r4/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r5]
I170719 18:40:41.290782 50313 sql/event_log.go:101  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170719 18:40:41.349143 50411 storage/replica_command.go:2673  [split,n1,s1,r5/1:/{Table/System���-Max}] initiating a split of this range at key /Table/11 [r6]
I170719 18:40:41.417176 50313 sql/lease.go:367  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-07-19 18:40:41.417022165 +0000 UTC
I170719 18:40:41.464897 50411 storage/replica_command.go:2673  [split,n1,s1,r6/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r7]
I170719 18:40:41.570660 50411 storage/replica_command.go:2673  [split,n1,s1,r7/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r8]
I170719 18:40:41.641366 50313 server/server.go:951  [n1] done ensuring all necessary migrations have run
I170719 18:40:41.642584 50313 server/server.go:953  [n1] serving sql connections
I170719 18:40:41.683176 50411 storage/replica_command.go:2673  [split,n1,s1,r8/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r9]
I170719 18:40:41.705747 50828 sql/event_log.go:101  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:60206} Attrs: Locality:} ClusterID:a185a4c7-e0ed-40fa-a7cd-3d3d21284208 StartedAt:1500489640942566356 LastUp:1500489640942566356}
I170719 18:40:41.818014 50411 storage/replica_command.go:2673  [split,n1,s1,r9/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r10]
W170719 18:40:42.013245 50313 server/server.go:299  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170719 18:40:42.026415 50313 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W170719 18:40:42.061493 50313 gossip/gossip.go:1196  [n?] no incoming or outgoing connections
I170719 18:40:42.088867 50313 server/config.go:534  [n?] 1 storage engine initialized
I170719 18:40:42.090230 50313 server/config.go:536  [n?] RocksDB cache size: 512 MiB
I170719 18:40:42.091016 50313 server/config.go:536  [n?] store 0: in-memory, size 100 MiB
I170719 18:40:42.093164 50313 server/node.go:434  [n?] store [n0,s0] not bootstrapped
I170719 18:40:42.093284 50313 storage/stores.go:295  [n?] read 0 node addresses from persistent storage
I170719 18:40:42.093533 50313 server/node.go:588  [n?] connecting to gossip network to verify cluster ID...
I170719 18:40:42.167213 50913 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:60206
I170719 18:40:42.169102 50974 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:38533}
I170719 18:40:42.172362 51000 storage/stores.go:314  [n?] wrote 1 node addresses to persistent storage
I170719 18:40:42.182534 50313 server/node.go:613  [n?] node connected via gossip and verified as part of cluster "a185a4c7-e0ed-40fa-a7cd-3d3d21284208"
I170719 18:40:42.199955 50313 kv/dist_sender.go:370  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170719 18:40:42.210559 50313 server/node.go:324  [n?] new node allocated ID 2
I170719 18:40:42.210920 50313 gossip/gossip.go:297  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:38533" > attrs:<> locality:<> 
I170719 18:40:42.211876 50313 server/node.go:385  [n2] node=2: started with [=] engine(s) and attributes []
I170719 18:40:42.227580 51013 storage/stores.go:314  [n1] wrote 1 node addresses to persistent storage
I170719 18:40:42.240254 50313 sql/executor.go:364  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:38533}
I170719 18:40:42.266098 50313 server/server.go:815  [n2] starting https server at 127.0.0.1:52863
I170719 18:40:42.266240 50313 server/server.go:816  [n2] starting grpc/postgres server at 127.0.0.1:38533
I170719 18:40:42.266396 50313 server/server.go:817  [n2] advertising CockroachDB node at 127.0.0.1:38533
I170719 18:40:42.282476 50313 server/server.go:951  [n2] done ensuring all necessary migrations have run
I170719 18:40:42.282663 50313 server/server.go:953  [n2] serving sql connections
I170719 18:40:42.291133 51008 server/node.go:569  [n2] bootstrapped store [n2,s2]
I170719 18:40:42.410001 51160 sql/event_log.go:101  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:38533} Attrs: Locality:} ClusterID:a185a4c7-e0ed-40fa-a7cd-3d3d21284208 StartedAt:1500489642211586320 LastUp:1500489642211586320}
W170719 18:40:42.430964 50313 server/server.go:299  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170719 18:40:42.442067 50313 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W170719 18:40:42.453081 50313 gossip/gossip.go:1196  [n?] no incoming or outgoing connections
I170719 18:40:42.530473 50313 server/config.go:534  [n?] 1 storage engine initialized
I170719 18:40:42.530584 50313 server/config.go:536  [n?] RocksDB cache size: 512 MiB
I170719 18:40:42.530619 50313 server/config.go:536  [n?] store 0: in-memory, size 100 MiB
I170719 18:40:42.532003 50313 server/node.go:434  [n?] store [n0,s0] not bootstrapped
I170719 18:40:42.532096 50313 storage/stores.go:295  [n?] read 0 node addresses from persistent storage
I170719 18:40:42.532213 50313 server/node.go:588  [n?] connecting to gossip network to verify cluster ID...
I170719 18:40:42.724378 51175 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:60206
I170719 18:40:42.726603 51252 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:58350}
I170719 18:40:42.740877 51238 storage/stores.go:314  [n?] wrote 1 node addresses to persistent storage
I170719 18:40:42.741205 50313 server/node.go:613  [n?] node connected via gossip and verified as part of cluster "a185a4c7-e0ed-40fa-a7cd-3d3d21284208"
I170719 18:40:42.747986 51239 storage/stores.go:314  [n?] wrote 2 node addresses to persistent storage
I170719 18:40:42.754604 50313 kv/dist_sender.go:370  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170719 18:40:42.760330 50313 server/node.go:324  [n?] new node allocated ID 3
I170719 18:40:42.760679 50313 gossip/gossip.go:297  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:58350" > attrs:<> locality:<> 
I170719 18:40:42.761468 50313 server/node.go:385  [n3] node=3: started with [=] engine(s) and attributes []
I170719 18:40:42.763950 50313 sql/executor.go:364  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:58350}
I170719 18:40:42.803862 51168 storage/stores.go:314  [n1] wrote 2 node addresses to persistent storage
I170719 18:40:42.806366 51210 storage/stores.go:314  [n2] wrote 2 node addresses to persistent storage
I170719 18:40:42.807662 50313 server/server.go:815  [n3] starting https server at 127.0.0.1:47309
I170719 18:40:42.807788 50313 server/server.go:816  [n3] starting grpc/postgres server at 127.0.0.1:58350
I170719 18:40:42.807836 50313 server/server.go:817  [n3] advertising CockroachDB node at 127.0.0.1:58350
I170719 18:40:42.814891 50313 server/server.go:951  [n3] done ensuring all necessary migrations have run
I170719 18:40:42.815060 50313 server/server.go:953  [n3] serving sql connections
I170719 18:40:42.898843 51211 sql/event_log.go:101  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:58350} Attrs: Locality:} ClusterID:a185a4c7-e0ed-40fa-a7cd-3d3d21284208 StartedAt:1500489642761147272 LastUp:1500489642761147272}
I170719 18:40:42.915698 51249 server/node.go:569  [n3] bootstrapped store [n3,s3]
I170719 18:40:42.924260 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r6/1:/Table/1{1-2}] generated preemptive snapshot 81e55aca at index 16
I170719 18:40:43.053543 50669 storage/store.go:3479  [replicate,n1,s1,r6/1:/Table/1{1-2}] streamed snapshot to (n2,s2):?: kv pairs: 10, log entries: 6, rate-limit: 8.0 MiB/sec, 5ms
I170719 18:40:43.054856 51431 storage/replica_raftstorage.go:705  [n2,s2,r6/?:{-}] applying preemptive snapshot at index 16 (id=81e55aca, encoded size=5443, 1 rocksdb batches, 6 log entries)
I170719 18:40:43.056503 51431 storage/replica_raftstorage.go:713  [n2,s2,r6/?:/Table/1{1-2}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170719 18:40:43.063141 50669 storage/replica_command.go:3606  [replicate,n1,s1,r6/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r6:/Table/1{1-2} [(n1,s1):1, next=2]
I170719 18:40:43.090168 51197 storage/replica.go:2947  [n1,s1,r6/1:/Table/1{1-2}] proposing ADD_REPLICA (n2,s2):2: [(n1,s1):1 (n2,s2):2]
I170719 18:40:43.103760 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r8/1:/Table/1{3-4}] generated preemptive snapshot a29c2bdc at index 26
I170719 18:40:43.141603 51434 storage/raft_transport.go:453  [n2] raft transport stream to node 1 established
I170719 18:40:43.268943 50669 storage/store.go:3479  [replicate,n1,s1,r8/1:/Table/1{3-4}] streamed snapshot to (n3,s3):?: kv pairs: 69, log entries: 16, rate-limit: 8.0 MiB/sec, 9ms
I170719 18:40:43.271393 51272 storage/replica_raftstorage.go:705  [n3,s3,r8/?:{-}] applying preemptive snapshot at index 26 (id=a29c2bdc, encoded size=21465, 1 rocksdb batches, 16 log entries)
I170719 18:40:43.276257 51272 storage/replica_raftstorage.go:713  [n3,s3,r8/?:/Table/1{3-4}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170719 18:40:43.292527 50669 storage/replica_command.go:3606  [replicate,n1,s1,r8/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r8:/Table/1{3-4} [(n1,s1):1, next=2]
I170719 18:40:43.390219 51514 storage/replica.go:2947  [n1,s1,r8/1:/Table/1{3-4}] proposing ADD_REPLICA (n3,s3):2: [(n1,s1):1 (n3,s3):2]
I170719 18:40:43.401263 51517 storage/raft_transport.go:453  [n3] raft transport stream to node 1 established
I170719 18:40:43.403979 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot 30b10ab4 at index 46
I170719 18:40:43.419086 50669 storage/store.go:3479  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n3,s3):?: kv pairs: 34, log entries: 36, rate-limit: 8.0 MiB/sec, 13ms
I170719 18:40:43.449252 51275 storage/replica_raftstorage.go:705  [n3,s3,r1/?:{-}] applying preemptive snapshot at index 46 (id=30b10ab4, encoded size=21450, 1 rocksdb batches, 36 log entries)
I170719 18:40:43.453086 51275 storage/replica_raftstorage.go:713  [n3,s3,r1/?:/{Min-System/}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170719 18:40:43.462641 50669 storage/replica_command.go:3606  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r1:/{Min-System/} [(n1,s1):1, next=2]
I170719 18:40:43.502298 51540 storage/replica.go:2947  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA (n3,s3):2: [(n1,s1):1 (n3,s3):2]
I170719 18:40:43.506769 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r3/1:/System/ts{d-e}] generated preemptive snapshot 91832340 at index 24
I170719 18:40:43.550884 50669 storage/store.go:3479  [replicate,n1,s1,r3/1:/System/ts{d-e}] streamed snapshot to (n2,s2):?: kv pairs: 911, log entries: 3, rate-limit: 8.0 MiB/sec, 43ms
I170719 18:40:43.552716 51542 storage/replica_raftstorage.go:705  [n2,s2,r3/?:{-}] applying preemptive snapshot at index 24 (id=91832340, encoded size=150432, 1 rocksdb batches, 3 log entries)
I170719 18:40:43.555772 51542 storage/replica_raftstorage.go:713  [n2,s2,r3/?:/System/ts{d-e}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170719 18:40:43.567043 50669 storage/replica_command.go:3606  [replicate,n1,s1,r3/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r3:/System/ts{d-e} [(n1,s1):1, next=2]
I170719 18:40:43.624562 51537 storage/replica.go:2947  [n1,s1,r3/1:/System/ts{d-e}] proposing ADD_REPLICA (n2,s2):2: [(n1,s1):1 (n2,s2):2]
I170719 18:40:43.628535 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] generated preemptive snapshot 3675bf7f at index 25
I170719 18:40:43.638838 50669 storage/store.go:3479  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] streamed snapshot to (n3,s3):?: kv pairs: 12, log entries: 15, rate-limit: 8.0 MiB/sec, 9ms
I170719 18:40:43.639680 51549 storage/replica_raftstorage.go:705  [n3,s3,r4/?:{-}] applying preemptive snapshot at index 25 (id=3675bf7f, encoded size=11686, 1 rocksdb batches, 15 log entries)
I170719 18:40:43.641407 51549 storage/replica_raftstorage.go:713  [n3,s3,r4/?:/{System/tse-Table/System���}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170719 18:40:43.646391 50669 storage/replica_command.go:3606  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r4:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, next=2]
I170719 18:40:43.756963 51621 storage/replica.go:2947  [n1,s1,r4/1:/{System/tse-Table/System���}] proposing ADD_REPLICA (n3,s3):2: [(n1,s1):1 (n3,s3):2]
I170719 18:40:43.764345 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] generated preemptive snapshot 6eeaa04a at index 27
I170719 18:40:43.787622 50669 storage/store.go:3479  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] streamed snapshot to (n2,s2):?: kv pairs: 40, log entries: 17, rate-limit: 8.0 MiB/sec, 21ms
I170719 18:40:43.789740 51551 storage/replica_raftstorage.go:705  [n2,s2,r5/?:{-}] applying preemptive snapshot at index 27 (id=6eeaa04a, encoded size=19426, 1 rocksdb batches, 17 log entries)
I170719 18:40:43.791715 51551 storage/replica_raftstorage.go:713  [n2,s2,r5/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170719 18:40:43.797140 50669 storage/replica_command.go:3606  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r5:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, next=2]
I170719 18:40:43.866899 51595 storage/replica.go:2947  [n1,s1,r5/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA (n2,s2):2: [(n1,s1):1 (n2,s2):2]
I170719 18:40:43.888076 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r9/1:/Table/1{4-5}] generated preemptive snapshot 938f0c8c at index 19
I170719 18:40:43.891766 50669 storage/store.go:3479  [replicate,n1,s1,r9/1:/Table/1{4-5}] streamed snapshot to (n3,s3):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 3ms
I170719 18:40:43.893571 51666 storage/replica_raftstorage.go:705  [n3,s3,r9/?:{-}] applying preemptive snapshot at index 19 (id=938f0c8c, encoded size=5870, 1 rocksdb batches, 9 log entries)
I170719 18:40:43.895901 51666 storage/replica_raftstorage.go:713  [n3,s3,r9/?:/Table/1{4-5}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170719 18:40:43.899156 50669 storage/replica_command.go:3606  [replicate,n1,s1,r9/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r9:/Table/1{4-5} [(n1,s1):1, next=2]
I170719 18:40:44.010797 51701 storage/replica.go:2947  [n1,s1,r9/1:/Table/1{4-5}] proposing ADD_REPLICA (n3,s3):2: [(n1,s1):1 (n3,s3):2]
I170719 18:40:44.017768 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r7/1:/Table/1{2-3}] generated preemptive snapshot 69186943 at index 25
I170719 18:40:44.093159 50669 storage/store.go:3479  [replicate,n1,s1,r7/1:/Table/1{2-3}] streamed snapshot to (n2,s2):?: kv pairs: 31, log entries: 15, rate-limit: 8.0 MiB/sec, 48ms
I170719 18:40:44.097616 51706 storage/replica_raftstorage.go:705  [n2,s2,r7/?:{-}] applying preemptive snapshot at index 25 (id=69186943, encoded size=16662, 1 rocksdb batches, 15 log entries)
I170719 18:40:44.099452 51706 storage/replica_raftstorage.go:713  [n2,s2,r7/?:/Table/1{2-3}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170719 18:40:44.105901 50669 storage/replica_command.go:3606  [replicate,n1,s1,r7/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r7:/Table/1{2-3} [(n1,s1):1, next=2]
I170719 18:40:44.160416 51612 storage/replica.go:2947  [n1,s1,r7/1:/Table/1{2-3}] proposing ADD_REPLICA (n2,s2):2: [(n1,s1):1 (n2,s2):2]
I170719 18:40:44.175010 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r10/1:/{Table/15-Max}] generated preemptive snapshot 5ea961de at index 11
I170719 18:40:44.178578 50669 storage/store.go:3479  [replicate,n1,s1,r10/1:/{Table/15-Max}] streamed snapshot to (n3,s3):?: kv pairs: 9, log entries: 1, rate-limit: 8.0 MiB/sec, 3ms
I170719 18:40:44.180008 51600 storage/replica_raftstorage.go:705  [n3,s3,r10/?:{-}] applying preemptive snapshot at index 11 (id=5ea961de, encoded size=548, 1 rocksdb batches, 1 log entries)
I170719 18:40:44.181387 51600 storage/replica_raftstorage.go:713  [n3,s3,r10/?:/{Table/15-Max}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170719 18:40:44.192858 50669 storage/replica_command.go:3606  [replicate,n1,s1,r10/1:/{Table/15-Max}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r10:/{Table/15-Max} [(n1,s1):1, next=2]
I170719 18:40:44.272391 51763 storage/replica.go:2947  [n1,s1,r10/1:/{Table/15-Max}] proposing ADD_REPLICA (n3,s3):2: [(n1,s1):1 (n3,s3):2]
I170719 18:40:44.276880 50669 storage/replica_raftstorage.go:496  [replicate,n1,s1,r2/1:/System/{-tsd}] generated preemptive snapshot 10b563e8 at index 40
I170719 18:40:44.292607 50669 storage/store.go:3479  [replicate,n1,s1,r2/1:/System/{-tsd}] streamed snapshot to (n2,s2):?: kv pairs: 32, log entries: 3, rate-limit: 8.0 MiB/sec, 12ms
I170719 18:40:44.294495 51665 storage/replica_raftstorage.go:705  [n2,s2,r2/?:{-}] applying preemptive snapshot at index 40 (id=10b563e8, encoded size=74635, 1 rocksdb batches, 3 log entries)
I170719 18:40:44.296820 51665 storage/replica_raftstorage.go:713  [n2,s2,r2/?:/System/{-tsd}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170719 18:40:44.300526 50669 storage/replica_command.go:3606  [replicate,n1,s1,r2/1:/System/{-tsd}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r2:/System/{-tsd} [(n1,s1):1, next=2]
I170719 18:40:44.368251 51737 storage/replica.go:2947  [n1,s1,r2/1:/System/{-tsd}] proposing ADD_REPLICA (n2,s2):2: [(n1,s1):1 (n2,s2):2]
I170719 18:40:44.377803 50669 storage/queue.go:725  [n1,replicate] purgatory is now empty
I170719 18:40:44.382176 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r6/1:/Table/1{1-2}] generated preemptive snapshot 4bea8438 at index 21
I170719 18:40:44.392734 50412 storage/store.go:3479  [replicate,n1,s1,r6/1:/Table/1{1-2}] streamed snapshot to (n3,s3):?: kv pairs: 12, log entries: 11, rate-limit: 8.0 MiB/sec, 8ms
I170719 18:40:44.401913 51768 storage/replica_raftstorage.go:705  [n3,s3,r6/?:{-}] applying preemptive snapshot at index 21 (id=4bea8438, encoded size=8431, 1 rocksdb batches, 11 log entries)
I170719 18:40:44.405612 51768 storage/replica_raftstorage.go:713  [n3,s3,r6/?:/Table/1{1-2}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170719 18:40:44.426634 50412 storage/replica_command.go:3606  [replicate,n1,s1,r6/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r6:/Table/1{1-2} [(n1,s1):1, (n2,s2):2, next=3]
I170719 18:40:44.522097 51756 storage/replica.go:2947  [n1,s1,r6/1:/Table/1{1-2}] proposing ADD_REPLICA (n3,s3):3: [(n1,s1):1 (n2,s2):2 (n3,s3):3]
I170719 18:40:44.538790 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r3/1:/System/ts{d-e}] generated preemptive snapshot dedc3551 at index 27
I170719 18:40:44.560014 50412 storage/store.go:3479  [replicate,n1,s1,r3/1:/System/ts{d-e}] streamed snapshot to (n3,s3):?: kv pairs: 912, log entries: 6, rate-limit: 8.0 MiB/sec, 20ms
I170719 18:40:44.583255 51678 storage/replica_raftstorage.go:705  [n3,s3,r3/?:{-}] applying preemptive snapshot at index 27 (id=dedc3551, encoded size=152532, 1 rocksdb batches, 6 log entries)
I170719 18:40:44.602153 51678 storage/replica_raftstorage.go:713  [n3,s3,r3/?:/System/ts{d-e}] applied preemptive snapshot in 19ms [clear=0ms batch=0ms entries=16ms commit=1ms]
I170719 18:40:44.610959 50412 storage/replica_command.go:3606  [replicate,n1,s1,r3/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r3:/System/ts{d-e} [(n1,s1):1, (n2,s2):2, next=3]
I170719 18:40:44.683592 51831 storage/replica.go:2947  [n1,s1,r3/1:/System/ts{d-e}] proposing ADD_REPLICA (n3,s3):3: [(n1,s1):1 (n2,s2):2 (n3,s3):3]
I170719 18:40:44.692645 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r8/1:/Table/1{3-4}] generated preemptive snapshot 1e9608b1 at index 51
I170719 18:40:44.699159 50412 storage/store.go:3479  [replicate,n1,s1,r8/1:/Table/1{3-4}] streamed snapshot to (n2,s2):?: kv pairs: 131, log entries: 41, rate-limit: 8.0 MiB/sec, 6ms
I170719 18:40:44.702296 51772 storage/replica_raftstorage.go:705  [n2,s2,r8/?:{-}] applying preemptive snapshot at index 51 (id=1e9608b1, encoded size=51779, 1 rocksdb batches, 41 log entries)
I170719 18:40:44.704948 51772 storage/replica_raftstorage.go:713  [n2,s2,r8/?:/Table/1{3-4}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170719 18:40:44.713856 50412 storage/replica_command.go:3606  [replicate,n1,s1,r8/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r8:/Table/1{3-4} [(n1,s1):1, (n3,s3):2, next=3]
I170719 18:40:44.775630 51803 storage/replica.go:2947  [n1,s1,r8/1:/Table/1{3-4}] proposing ADD_REPLICA (n2,s2):3: [(n1,s1):1 (n3,s3):2 (n2,s2):3]
I170719 18:40:44.808240 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r7/1:/Table/1{2-3}] generated preemptive snapshot 5b1246fb at index 28
I170719 18:40:44.814480 50412 storage/store.go:3479  [replicate,n1,s1,r7/1:/Table/1{2-3}] streamed snapshot to (n3,s3):?: kv pairs: 32, log entries: 18, rate-limit: 8.0 MiB/sec, 5ms
I170719 18:40:44.816750 51878 storage/replica_raftstorage.go:705  [n3,s3,r7/?:{-}] applying preemptive snapshot at index 28 (id=5b1246fb, encoded size=18693, 1 rocksdb batches, 18 log entries)
I170719 18:40:44.820136 51878 storage/replica_raftstorage.go:713  [n3,s3,r7/?:/Table/1{2-3}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170719 18:40:44.831633 50412 storage/replica_command.go:3606  [replicate,n1,s1,r7/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r7:/Table/1{2-3} [(n1,s1):1, (n2,s2):2, next=3]
I170719 18:40:44.909251 51909 storage/replica.go:2947  [n1,s1,r7/1:/Table/1{2-3}] proposing ADD_REPLICA (n3,s3):3: [(n1,s1):1 (n2,s2):2 (n3,s3):3]
I170719 18:40:44.920047 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] generated preemptive snapshot b72fcd3d at index 30
I170719 18:40:44.933548 50412 storage/store.go:3479  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] streamed snapshot to (n3,s3):?: kv pairs: 41, log entries: 20, rate-limit: 8.0 MiB/sec, 11ms
I170719 18:40:44.935452 51924 storage/replica_raftstorage.go:705  [n3,s3,r5/?:{-}] applying preemptive snapshot at index 30 (id=b72fcd3d, encoded size=21457, 1 rocksdb batches, 20 log entries)
I170719 18:40:44.941615 51924 storage/replica_raftstorage.go:713  [n3,s3,r5/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170719 18:40:44.955497 50412 storage/replica_command.go:3606  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r5:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, (n2,s2):2, next=3]
I170719 18:40:45.029256 51918 storage/replica.go:2947  [n1,s1,r5/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA (n3,s3):3: [(n1,s1):1 (n2,s2):2 (n3,s3):3]
I170719 18:40:45.040591 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r2/1:/System/{-tsd}] generated preemptive snapshot a3676c17 at index 43
I170719 18:40:45.060519 50412 storage/store.go:3479  [replicate,n1,s1,r2/1:/System/{-tsd}] streamed snapshot to (n3,s3):?: kv pairs: 33, log entries: 6, rate-limit: 8.0 MiB/sec, 15ms
I170719 18:40:45.083606 51863 storage/replica_raftstorage.go:705  [n3,s3,r2/?:{-}] applying preemptive snapshot at index 43 (id=a3676c17, encoded size=76678, 1 rocksdb batches, 6 log entries)
I170719 18:40:45.144547 51863 storage/replica_raftstorage.go:713  [n3,s3,r2/?:/System/{-tsd}] applied preemptive snapshot in 61ms [clear=0ms batch=29ms entries=27ms commit=3ms]
I170719 18:40:45.156995 50412 storage/replica_command.go:3606  [replicate,n1,s1,r2/1:/System/{-tsd}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r2:/System/{-tsd} [(n1,s1):1, (n2,s2):2, next=3]
I170719 18:40:45.228832 51884 storage/replica.go:2947  [n1,s1,r2/1:/System/{-tsd}] proposing ADD_REPLICA (n3,s3):3: [(n1,s1):1 (n2,s2):2 (n3,s3):3]
I170719 18:40:45.240561 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] generated preemptive snapshot b2348277 at index 28
I170719 18:40:45.255761 50412 storage/store.go:3479  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] streamed snapshot to (n2,s2):?: kv pairs: 13, log entries: 18, rate-limit: 8.0 MiB/sec, 11ms
I170719 18:40:45.257762 51988 storage/replica_raftstorage.go:705  [n2,s2,r4/?:{-}] applying preemptive snapshot at index 28 (id=b2348277, encoded size=13774, 1 rocksdb batches, 18 log entries)
I170719 18:40:45.260475 51988 storage/replica_raftstorage.go:713  [n2,s2,r4/?:/{System/tse-Table/System���}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170719 18:40:45.266517 50412 storage/replica_command.go:3606  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r4:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, (n3,s3):2, next=3]
I170719 18:40:45.336241 51959 storage/replica.go:2947  [n1,s1,r4/1:/{System/tse-Table/System���}] proposing ADD_REPLICA (n2,s2):3: [(n1,s1):1 (n3,s3):2 (n2,s2):3]
I170719 18:40:45.357539 51386 storage/replica_proposal.go:449  [n3,s3,r6/3:/Table/1{1-2}] new range lease repl=(n3,s3):3 start=1500489645.346104402,0 epo=1 pro=1500489645.346113302,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:45.363770 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot ecbcb4c9 at index 78
I170719 18:40:45.369321 50412 storage/store.go:3479  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n2,s2):?: kv pairs: 50, log entries: 68, rate-limit: 8.0 MiB/sec, 5ms
I170719 18:40:45.372201 52006 storage/replica_raftstorage.go:705  [n2,s2,r1/?:{-}] applying preemptive snapshot at index 78 (id=ecbcb4c9, encoded size=37979, 1 rocksdb batches, 68 log entries)
I170719 18:40:45.377124 52006 storage/replica_raftstorage.go:713  [n2,s2,r1/?:/{Min-System/}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170719 18:40:45.399349 50412 storage/replica_command.go:3606  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r1:/{Min-System/} [(n1,s1):1, (n3,s3):2, next=3]
I170719 18:40:45.481218 51871 storage/replica.go:2947  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA (n2,s2):3: [(n1,s1):1 (n3,s3):2 (n2,s2):3]
I170719 18:40:45.507428 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r10/1:/{Table/15-Max}] generated preemptive snapshot 146490fd at index 16
I170719 18:40:45.511341 50412 storage/store.go:3479  [replicate,n1,s1,r10/1:/{Table/15-Max}] streamed snapshot to (n2,s2):?: kv pairs: 11, log entries: 6, rate-limit: 8.0 MiB/sec, 3ms
I170719 18:40:45.513151 51993 storage/replica_raftstorage.go:705  [n2,s2,r10/?:{-}] applying preemptive snapshot at index 16 (id=146490fd, encoded size=3542, 1 rocksdb batches, 6 log entries)
I170719 18:40:45.516354 51993 storage/replica_raftstorage.go:713  [n2,s2,r10/?:/{Table/15-Max}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=0ms commit=2ms]
I170719 18:40:45.539440 50412 storage/replica_command.go:3606  [replicate,n1,s1,r10/1:/{Table/15-Max}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r10:/{Table/15-Max} [(n1,s1):1, (n3,s3):2, next=3]
I170719 18:40:45.594948 52021 storage/replica.go:2947  [n1,s1,r10/1:/{Table/15-Max}] proposing ADD_REPLICA (n2,s2):3: [(n1,s1):1 (n3,s3):2 (n2,s2):3]
I170719 18:40:45.626592 50412 storage/replica_raftstorage.go:496  [replicate,n1,s1,r9/1:/Table/1{4-5}] generated preemptive snapshot 2eeed046 at index 24
I170719 18:40:45.675668 52046 storage/replica_raftstorage.go:705  [n2,s2,r9/?:{-}] applying preemptive snapshot at index 24 (id=2eeed046, encoded size=8858, 1 rocksdb batches, 14 log entries)
I170719 18:40:45.677265 52046 storage/replica_raftstorage.go:713  [n2,s2,r9/?:/Table/1{4-5}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170719 18:40:45.678460 50412 storage/store.go:3479  [replicate,n1,s1,r9/1:/Table/1{4-5}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 14, rate-limit: 8.0 MiB/sec, 45ms
I170719 18:40:45.682760 52056 storage/raft_transport.go:453  [n3] raft transport stream to node 2 established
I170719 18:40:45.697634 50412 storage/replica_command.go:3606  [replicate,n1,s1,r9/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r9:/Table/1{4-5} [(n1,s1):1, (n3,s3):2, next=3]
I170719 18:40:45.781228 52125 storage/replica.go:2947  [n1,s1,r9/1:/Table/1{4-5}] proposing ADD_REPLICA (n2,s2):3: [(n1,s1):1 (n3,s3):2 (n2,s2):3]
I170719 18:40:45.810123 52065 storage/raft_transport.go:453  [n2] raft transport stream to node 3 established
I170719 18:40:46.145881 50411 storage/replica_command.go:2673  [split,n1,s1,r10/1:/{Table/15-Max}] initiating a split of this range at key /Table/50 [r11]
I170719 18:40:46.154443 52104 sql/event_log.go:101  [client=127.0.0.1:42049,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:data Statement:CREATE DATABASE IF NOT EXISTS data User:root}
I170719 18:40:46.335190 52104 sql/event_log.go:101  [client=127.0.0.1:42049,user=root,n1] Event: "create_table", target: 51, info: {TableName:data.bank Statement:CREATE TABLE data.bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I170719 18:40:46.388780 50411 storage/replica_command.go:2673  [split,n1,s1,r11/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r12]
I170719 18:40:46.721514 51036 storage/replica_proposal.go:449  [n2,s2,r11/3:/Table/5{0-1}] new range lease repl=(n2,s2):3 start=1500489646.656808267,0 epo=1 pro=1500489646.672018359,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:47.583193 52344 storage/replica_command.go:2673  [n1,s1,r12/1:/{Table/51-Max}] initiating a split of this range at key /Table/51/1/0 [r13]
I170719 18:40:47.764403 52388 storage/replica_command.go:2673  [n1,s1,r13/1:/{Table/51/1/0-Max}] initiating a split of this range at key /Table/51/1/1 [r14]
I170719 18:40:48.024040 52437 storage/replica_command.go:2673  [n1,s1,r14/1:/{Table/51/1/1-Max}] initiating a split of this range at key /Table/51/1/2 [r15]
I170719 18:40:48.206567 52456 storage/replica_command.go:2673  [n1,s1,r15/1:/{Table/51/1/2-Max}] initiating a split of this range at key /Table/51/1/3 [r16]
I170719 18:40:48.399966 52522 storage/replica_command.go:2673  [n1,s1,r16/1:/{Table/51/1/3-Max}] initiating a split of this range at key /Table/51/1/4 [r17]
I170719 18:40:48.689980 52567 storage/replica_command.go:2673  [n1,s1,r17/1:/{Table/51/1/4-Max}] initiating a split of this range at key /Table/51/1/5 [r18]
I170719 18:40:48.753295 51089 storage/replica_proposal.go:449  [n2,s2,r10/3:/Table/{15-50}] new range lease repl=(n2,s2):3 start=1500489648.736211591,0 epo=1 pro=1500489648.736220891,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:48.899607 52602 storage/replica_command.go:2673  [n1,s1,r18/1:/{Table/51/1/5-Max}] initiating a split of this range at key /Table/51/1/6 [r19]
I170719 18:40:49.137039 52666 storage/replica_command.go:2673  [n1,s1,r19/1:/{Table/51/1/6-Max}] initiating a split of this range at key /Table/51/1/7 [r20]
I170719 18:40:49.318726 52703 storage/replica_command.go:2673  [n1,s1,r20/1:/{Table/51/1/7-Max}] initiating a split of this range at key /Table/51/1/8 [r21]
I170719 18:40:49.463074 50551 storage/replica_proposal.go:449  [n1,s1,r20/1:/{Table/51/1/7-Max}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.456884527,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:49.512239 50565 storage/replica_proposal.go:449  [n1,s1,r20/1:/{Table/51/1/7-Max}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.456884527,0 following repl=(n0,s0):? start=0.000000000,0 exp=0.000000000,0
I170719 18:40:49.539785 50540 storage/replica_proposal.go:449  [n1,s1,r8/1:/Table/1{3-4}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.530197735,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:49.642566 50554 storage/replica_proposal.go:449  [n1,s1,r5/1:/Table/{SystemCon���-11}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.631295676,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:49.710226 50528 storage/replica_proposal.go:449  [n1,s1,r7/1:/Table/1{2-3}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.700723509,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:49.761233 50411 storage/replica_command.go:2673  [split,n1,s1,r21/1:/{Table/51/1/8-Max}] initiating a split of this range at key /Table/52 [r22]
I170719 18:40:49.771369 52104 sql/event_log.go:101  [client=127.0.0.1:42049,user=root,n1] Event: "create_database", target: 52, info: {DatabaseName:restoredb Statement:CREATE DATABASE restoredb User:root}
I170719 18:40:50.014066 51368 storage/replica_proposal.go:449  [replica consistency checker,n3,s3,r17/2:/Table/51/1/{4-5}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489649.996621290,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.072598 50525 storage/replica_proposal.go:449  [n1,s1,r21/1:/{Table/51/1/8-Max}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489649.456884527,0 following repl=(n0,s0):? start=0.000000000,0 exp=0.000000000,0
I170719 18:40:50.486173 50514 storage/replica_proposal.go:449  [replica consistency checker,n1,s1,r18/1:/Table/51/1/{5-6}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1500489650.480133073,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.540946 53139 ccl/storageccl/export.go:129  [n1,s1,r5/1:/Table/{SystemCon���-11}] export [/Table/4/1,/Table/4/2)
I170719 18:40:50.546221 53124 ccl/storageccl/export.go:129  [n3,s3,r17/2:/Table/51/1/{4-5}] export [/Table/51/1/4,/Table/51/1/5)
I170719 18:40:50.550387 53138 ccl/storageccl/export.go:129  [n1,s1,r5/1:/Table/{SystemCon���-11}] export [/Table/3/1,/Table/3/2)
I170719 18:40:50.565638 53125 ccl/storageccl/export.go:129  [n1,s1,r21/1:/Table/5{1/1/8-2}] export [/Table/51/1/8,/Table/51/2)
I170719 18:40:50.570950 53143 ccl/storageccl/export.go:129  [n1,s1,r20/1:/Table/51/1/{7-8}] export [/Table/51/1/7,/Table/51/1/8)
I170719 18:40:50.578009 53031 ccl/storageccl/export.go:129  [n1,s1,r18/1:/Table/51/1/{5-6}] export [/Table/51/1/5,/Table/51/1/6)
I170719 18:40:50.579386 51342 storage/replica_proposal.go:449  [n3,s3,r14/2:/Table/51/1/{1-2}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.545436627,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.584167 51405 storage/replica_proposal.go:449  [n3,s3,r15/2:/Table/51/1/{2-3}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.546828454,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.616689 51375 storage/replica_proposal.go:449  [n3,s3,r12/2:/Table/51{-/1/0}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.556122132,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.618563 51396 storage/replica_proposal.go:449  [n3,s3,r19/2:/Table/51/1/{6-7}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.569434188,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.620491 51382 storage/replica_proposal.go:449  [n3,s3,r13/2:/Table/51/1/{0-1}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.561660439,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:50.624096 51380 storage/replica_proposal.go:449  [n3,s3,r16/2:/Table/51/1/{3-4}] new range lease repl=(n3,s3):2 start=1500489649.918328190,1 epo=1 pro=1500489650.573265961,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1500489649.918328190,0 pro=1500489640.918384591,0
I170719 18:40:51.055861 53186 ccl/storageccl/export.go:129  [n3,s3,r12/2:/Table/51{-/1/0}] export [/Table/51/1,/Table/51/1/0)
I170719 18:40:51.056260 53187 ccl/storageccl/export.go:129  [n3,s3,r13/2:/Table/51/1/{0-1}] export [/Table/51/1/0,/Table/51/1/1)
I170719 18:40:51.056375 53190 ccl/storageccl/export.go:129  [n3,s3,r19/2:/Table/51/1/{6-7}] export [/Table/51/1/6,/Table/51/1/7)
I170719 18:40:51.057016 53191 ccl/storageccl/export.go:129  [n3,s3,r16/2:/Table/51/1/{3-4}] export [/Table/51/1/3,/Table/51/1/4)
I170719 18:40:51.058199 53028 ccl/storageccl/export.go:129  [n3,s3,r15/2:/Table/51/1/{2-3}] export [/Table/51/1/2,/Table/51/1/3)
==================
Write at 0x00c420b79c90 by goroutine 1159:
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.Backup.func3()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:463 +0x651
  github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1()
      /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:58 +0x68

Previous read at 0x00c420b79c90 by goroutine 1059:
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.Backup.func3()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:487 +0xa10
  github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1()
      /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:58 +0x68

Goroutine 1159 (running) created at:
  github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go()
      /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:66 +0x73
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.Backup()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:498 +0x16bb
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.backupPlanHook.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:606 +0x72b
  github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).Start()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:75 +0x65
  github.com/cockroachdb/cockroach/pkg/sql.(*planner).startPlan()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/plan.go:209 +0xb5
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execClassic()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1572 +0x20a
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmt()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1715 +0x8b4
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmtInOpenTxn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1436 +0xc60
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmtsInCurrentTxn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1059 +0xb8e
  github.com/cockroachdb/cockroach/pkg/sql.runTxnAttempt()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:968 +0xbf
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execParsed.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:773 +0x2bd
  github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Exec()
      /go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:675 +0xe2
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execParsed()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:795 +0x6cc
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execPrepared()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:593 +0xbb
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).ExecutePreparedStatement()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:575 +0x233
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).handleExecute()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:841 +0x32b
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:463 +0xacd
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:421 +0xe7c
  github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func8.1()
      /go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:687 +0x17e
  github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:142 +0xdc

Goroutine 1059 (finished) created at:
  github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go()
      /go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:66 +0x73
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.Backup()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:498 +0x16bb
  github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.backupPlanHook.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup.go:606 +0x72b
  github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).Start()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:75 +0x65
  github.com/cockroachdb/cockroach/pkg/sql.(*planner).startPlan()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/plan.go:209 +0xb5
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execClassic()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1572 +0x20a
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmt()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1715 +0x8b4
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmtInOpenTxn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1436 +0xc60
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execStmtsInCurrentTxn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:1059 +0xb8e
  github.com/cockroachdb/cockroach/pkg/sql.runTxnAttempt()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:968 +0xbf
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execParsed.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:773 +0x2bd
  github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Exec()
      /go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:675 +0xe2
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execParsed()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:795 +0x6cc
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).execPrepared()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:593 +0xbb
  github.com/cockroachdb/cockroach/pkg/sql.(*Executor).ExecutePreparedStatement()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/executor.go:575 +0x233
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).handleExecute()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:841 +0x32b
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:463 +0xacd
  github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn()
      /go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:421 +0xe7c
  github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func8.1()
      /go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:687 +0x17e
  github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1()
      /go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:142 +0xdc
==================
I170719 18:38:12.036696 1 rand.go:76  Random seed: 4606219213896658469

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/ccl/sqlccl/backup.go
+++ b/pkg/ccl/sqlccl/backup.go
@@ -48,10 +48,11 @@ const (
 	BackupDescriptorCheckpointName = "BACKUP-CHECKPOINT"
 	// BackupFormatInitialVersion is the first version of backup and its files.
 	BackupFormatInitialVersion uint32 = 0
-	// backupCheckpointInterval is the interval at which backup progress is saved
-	// to durable storage.
-	backupCheckpointInterval = time.Minute
 )
+
+// BackupCheckpointInterval is the interval at which backup progress is saved
+// to durable storage.
+var BackupCheckpointInterval = time.Minute
 
 // exportStorageFromURI returns an ExportStorage for the given URI.
 func exportStorageFromURI(ctx context.Context, uri string) (storageccl.ExportStorage, error) {
@@ -467,11 +468,12 @@ func Backup(
 				})
 				mu.exported.Add(file.Exported)
 			}
-			doCheckpoint := timeutil.Since(mu.lastCheckpoint) > backupCheckpointInterval
-			if doCheckpoint {
+			var checkpointFiles backupFileDescriptors
+			if timeutil.Since(mu.lastCheckpoint) > BackupCheckpointInterval {
 				// We optimistically assume the checkpoint will succeed to prevent
 				// multiple threads from attempting to checkpoint.
 				mu.lastCheckpoint = timeutil.Now()
+				checkpointFiles = append(checkpointFiles, mu.files...)
 			}
 			mu.Unlock()
 
@@ -482,9 +484,9 @@ func Backup(
 					jobLogger.JobID(), jobLogger.Job.Description, err)
 			}
 
-			if doCheckpoint {
+			if checkpointFiles != nil {
 				desc.Lock()
-				desc.Files = append(backupFileDescriptors(nil), mu.files...)
+				desc.Files = checkpointFiles
 				if err := writeBackupDescriptor(
 					ctx, exportStore, BackupDescriptorCheckpointName, &desc.BackupDescriptor,
 				); err != nil {

--- a/pkg/ccl/sqlccl/backup_test.go
+++ b/pkg/ccl/sqlccl/backup_test.go
@@ -25,6 +25,7 @@ import (
 	"strings"
 	"sync/atomic"
 	"testing"
+	"time"
 
 	"github.com/kr/pretty"
 	"github.com/lib/pq"
@@ -589,6 +590,11 @@ func TestBackupRestoreSystemJobsProgress(t *testing.T) {
 
 func TestBackupRestoreCheckpointing(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+
+	defer func(oldInterval time.Duration) {
+		sqlccl.BackupCheckpointInterval = oldInterval
+	}(sqlccl.BackupCheckpointInterval)
+	sqlccl.BackupCheckpointInterval = time.Millisecond
 
 	var checkpointPath string
 

